pmlc/rt/device_id.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Intel Corporation
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "pmlc/rt/runtime.h"
+#include "llvm/ADT/StringRef.h"
+
+namespace pmlc::runtime {
```

Comment:
The dir structure and namespace hierarchy should match

Reply:
That bothered me, as well; I figured it could go in a separate PR, but, done.

Reply:
Use `pmlc::rt`